Formula/traefik.rb

```diff
@@ -0,0 +1,70 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "Træfik, a modern reverse proxy"
```

Comment:
There's an audit rule against starting the description with the formula name, which you've managed to circumvent, but still applies :)

Reply:
Heh)
Well, that's what they have in description of github repository.
What should i write?
Just "A modern reverse proxy"?

Reply:
Pushed with that.
By the way, where can i find those audit rules?

Reply:
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb
https://github.com/Homebrew/brew/tree/master/Library/Homebrew/rubocops

in general just

brew audit --new-formula traefik